scripts/check_backend_associated_test_file.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2022 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script checks if every backend file has an associated test file."""
+
+from __future__ import annotations
+
+import logging
+import os
+import sys
+
+TOPMOST_LEVEL_PATH = './'
+
+# NOTE TO DEVELOPERS: Do not add any new files to this list without asking
+# @U8NWXD first.
```

Comment:
```suggestion
# the Dev Workflow Team first.
```

Just to future-proof in case someone else takes over leading the team

Reply:
Updated it now.